src/controllers/itemDetails/index.js

```diff
@@ -1031,6 +1031,29 @@ function renderDirector(page, item, context) {
     }
 }
 
+function renderStudio(page, item, context) {
+    const studios = item.Studios || [];
+
+    const html = studios.map(function (studio) {
+        return '<a style="color:inherit;" class="button-link" is="emby-linkbutton" href="' + appRouter.getRouteUrl({
+            Name: studio.Name,
+            Type: 'Studio',
+            ServerId: item.ServerId,
+            Id: studio.Id
+        }, {
+            context: context
+        }) + '">' + escapeHtml(studio.Name) + '</a>';
+    }).join(', ');
+
+    const studiosLabel = page.querySelector('.studiosLabel');
+    studiosLabel.innerHTML = globalize.translate(studios.length > 1 ? 'Studios' : 'Studio');
```

Comment:
`innerText` is safer to use if you are only inserting text (removes the possibility of inadvertently introducing a cross-site scripting vulnerability)

```suggestion
    studiosLabel.innerText = globalize.translate(studios.length > 1 ? 'Studios' : 'Studio');
```